components/kyma-environment-broker/internal/auditlog/config.go

```diff
@@ -6,4 +6,5 @@ type Config struct {
 	Password      string `envconfig:"APP_AUDITLOG_PASSWORD"`
 	Tenant        string `envconfig:"APP_AUDITLOG_TENANT"`
 	EnableSeqHttp bool   `envconfig:"APP_AUDITLOG_ENABLE_SEQ_HTTP"`
+	Disabled      bool   `envconfig:"default=false"`
```

Comment:
This annotation looks different from the rest in the struct. Will it work?

Reply:
The `envconfig` accepts default value: https://github.com/vrischmann/envconfig#default-values